docs/r2dbc.md

```diff
@@ -203,16 +206,19 @@ the supplied service account. The IAM principal must have the
 iam.serviceAccounts.getAccessToken permission or the role
 roles/iam.serviceAccounts.serviceAccountTokenCreator.
 
-For example, if the full IAM user account
-is `my-sa@my-project.iam.gserviceaccount.com`,
-You would update the code, replacing these examples with the appropriate values
-for your database engine:
+You must enable IAM Authentication to use service account impersonation.
+Set the `TARGET_PRINCIPAL` property to the full IAM email. Set
+the `USER` option to the shortened IAM email following the rules described
+in [IAM Authentication](#iam-authentication)
+
+#### Example
+
+Replace these parameters in the example based on your database type:
 
-|               | Mysql                                    | Postgres                                 |
-|---------------|------------------------------------------|------------------------------------------|
-| <PROTOCOL>    | mysql                                    | postgresql                               |  
-| <IAM_DB_USER> | my-sa                                    | my-sa@my-proeject.iam                    |
-| <IAM_EMAIL>   | my-sa@my-project.iam.gserviceaccount.com | my-sa@my-project.iam.gserviceaccount.com |
+|          | PROTOCOL   | IAM_DB_USER          | IAM_EMAIL                                |
+|----------|------------|----------------------|------------------------------------------|
+| Mysql    | mysql      | my-sa                | my-sa@my-project.iam.gserviceaccount.com |  
```

Comment:
With the protocol column, do we need the extra MySQL and Postgres column?

Reply:
It's not strictly required, but I think it improves the clarity of the documentation.